StyleCop.Analyzers/StyleCop.Analyzers/NamingRules/SX1309FieldNamesMustBeginWithUnderscore.cs

```diff
@@ -33,10 +33,10 @@ internal class SX1309FieldNamesMustBeginWithUnderscore : DiagnosticAnalyzer
         /// The ID for diagnostics produced by the <see cref="SX1309FieldNamesMustBeginWithUnderscore"/> analyzer.
         /// </summary>
         public const string DiagnosticId = "SX1309";
-        private const string Title = "Field names should begin with underscore";
-        private const string MessageFormat = "Field '{0}' should begin with an underscore";
-        private const string Description = "A field name in C# does not begin with an underscore.";
```

Comment:
❗️ These were incorrectly translated (the values were lost when SX1309S was translated to the same strings)